package.json

```diff
@@ -1,10 +1,13 @@
 {
   "name": "course.thesis",
   "version": "1.0.0",
+  "engines": {
+    "npm": "5.3.0",
+    "node": "8.2.1"
+  },
   "scripts": {
-    "postinstall": "curl -s https://raw.githubusercontent.com/reactorcore/pomander/master/bin/install | bash",
     "start": "nodemon server",
-    "test": "NODE_ENV=test grunt test",
+    "test": "NODE_ENV=production",
```

Comment:
You are setting npm run test to simply set the NODE_ENV to production. Why are we doing this here?

Reply:
My travis CI builds were failing because of an invalid Node ENV.
![screen shot 2017-09-05 at 2 45 02 pm](https://user-images.githubusercontent.com/24797620/30084827-def035a0-9248-11e7-8f78-1788e822ea93.png)

Reply:
Gotcha! Yea I would instead pass the env variable in the Gruntfile. https://github.com/nickelow/chipperchalices/blob/c350937d40ada6b79d9a834649592d1fbf1782f1/Gruntfile.js#L12